weave

```diff
@@ -115,10 +115,35 @@ docker_run_options() {
     echo --privileged --net=host $(docker_sock_options)
 }
 
+docker_exec_cni_options() {
```

Comment:
Note I needed the same `-v` flags in the CNI mini-script.
See 88cb07203aeb7d1fbe2e7d97b33a68b41719b2a7